### DISCUSSION TITLE
Coinbase Portfolio CSV Import

### DISCUSSION THREAD

**Discussion by kiesekae** [ghostfolio/ghostfolio#1927] [Ideas]
I stumbled upon this and thought it looked like a great!

Currently trying to import some of my Coinbase transactions with a CSV but keep getting all sorts of errors. I have worked though a few of them but I am stuck on `activities.0.date is not valid` : `"Date": "2020-11-16T23:29:45.000Z",`

I tried exporting a test activity from Ghostfolio to get an idea of a proper CSV template and Date Format so I could copy it to the Coinbase export but I was only able to figure out how to export a JSON from Ghostfolio. 

Any tips or points in the right direction would be appriciated!

**Comment by dtslvr** [comment]
Hi @kiesekae

You can find a few example files (`CSV` and `JSON`) [here](https://github.com/ghostfolio/ghostfolio/tree/main/test/import).

**Reply by kiesekae** [reply]
Thanks for your reply and I apologize for any ignorance on my part but;

It looks like the date format that Coinbase generates during an export is the accepted format shown in the [ok.json](https://github.com/ghostfolio/ghostfolio/blob/main/test/import/ok.json) example but not in the [ok.csv](https://github.com/ghostfolio/ghostfolio/blob/main/test/import/ok.csv) example.

I don't see data for Time of Day in the CSV example. Is it not possible to import a CSV with this date format?: `2020-11-16T23:29:45Z`

**Comment by leikoilja** [comment]
hey @kiesekae. have you figured out Coinbase to Ghostfolio export?
